internal/service/alarms/api/openapi.yaml

```diff
@@ -576,6 +677,23 @@ components:
         perceivedSeverity:
             $ref: '#/components/schemas/PerceivedSeverity'
 
+    AlarmServiceConfiguration:
+      type: object
+      properties:
+        retentionPeriod:
+          type: integer
+          description: | 
+            Number of days for alarm history to be retained.
+            This value has cannot be set lower than 1 (day).
+          example: 30
```

Comment:
Not sure how flexible we want to be...consider something similar to to say Prometheuse retention period. So allow user to put something like `30d` or `24h`.

But if settling with int..try hour instead of day...that we can simply use the time.ParseDuration to get the interval.

Reply:
This is set by the specification. I mean, that the value represent days. They do not mention what the minimum is, just that there is a minimum. I'm proposing 1 (day)

Reply:
ah then all good